python/lsst/daf/butler/core/registry.py

```diff
@@ -667,10 +697,16 @@ def findDataUnitEntry(self, dataUnitName, value):
         dataUnitEntry : `dict`
             Dictionary with all `DataUnit` values, or `None` if no matching
             entry is found.
+
+        Raises
+        ------
+        NotImplementedError
+            If ``self.limited`` is ``True``.
```

Comment:
There must be a way to make this link to the classes `limited` property in the documentation. @jonathansick ?

Reply:
I'd try

```
If `limited` is `True`.
```

If that doesn't work, I'd do

```
If `RegistryConfig.limited` is `True`.
```

Reply:
Also, the recommended way of writing the sentence is:

```
Raised if `RegistryConfig.limited` is `True`.

https://developer.lsst.io/python/numpydoc.html#py-docstring-raises

Reply:
Looks like it works if I just say
```
    If `limited` is `True`.
```